src/Components/Onboarding/Wizard.tsx

```diff
@@ -14,11 +15,36 @@ interface State {
 class Wizard extends React.Component<Props, State> {
   static defaultProps = {
     redirectTo: "/",
+    forceStep: "",
```

Comment:
Should the default value be `null` or an empty string?

Reply:
Ah good catch, this should be null.